hydra/_internal/utils.py

```diff
@@ -501,7 +501,14 @@ def __repr__(self) -> str:
         help="Adds an additional config dir to the config search path",
     )
 
-    info_choices = ["all", "defaults", "defaults-tree", "config", "plugins"]
+    info_choices = [
+        "all",
+        "defaults",
+        "defaults-tree",
+        "config",
+        "plugins",
+        "searchpath",
+    ]
```

Comment:
This is almost sorted, can you fix it and add a comment to keep it sorted?

Reply:
fixed the ordering. will merge after the CI pass.